test/e2e/apps/job.go

```diff
@@ -71,23 +71,16 @@ type watchEventConfig struct {
 var _ = SIGDescribe("Job", func() {
 	f := framework.NewDefaultFramework("job")
 	f.NamespacePodSecurityLevel = admissionapi.LevelPrivileged
-	parallelism := int32(2)
-	completions := int32(4)
-
-	largeParallelism := int32(90)
-	largeCompletions := int32(90)
-
-	backoffLimit := int32(6) // default value
 
 	// Simplest case: N pods succeed
 	ginkgo.It("should run a job to completion when tasks succeed", func(ctx context.Context) {
 		ginkgo.By("Creating a job")
-		job := e2ejob.NewTestJob("succeed", "all-succeed", v1.RestartPolicyNever, parallelism, completions, nil, backoffLimit)
+		job := e2ejob.NewTestJob("succeed", "all-succeed", v1.RestartPolicyNever, int32(2), int32(4), nil, int32(6))
```

Comment:
I would suggest to still keep the needed parameters named locally. Two reasons,
- when passed to the function it is not clear what they mean
- the completions =4 needs to match with the line below `err = e2ejob.WaitForJobComplete(ctx, f.ClientSet, f.Namespace.Name, job.Name, int32(4))`.

Reply:
fixed. PTAL :)

Reply:
Thanks, but the build fails currently, please fix.

Reply:
ci passed, PTAL again, thx :)

Reply:
lgtm

Reply:
> ci passed, PTAL again, thx :) cc @soltysh

@HuSharp I wanted to understand what it means by "CI passed" in this case. did you run "go test" locally to ensure your changing were successful? - is that what you mean by "CI passed" or is there a CI pipeline that we can leverage for such verifications before code merge? Trying to educate myself. Thanks in advance.